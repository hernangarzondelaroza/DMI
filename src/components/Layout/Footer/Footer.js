import React from "react";
import { Row } from "react-bootstrap";

import "./Footer.scss";

const footer = () => {
  return (
    <Row>
      <footer className="footer fixed-bottom">
        by Hernan Garzon de la Roza
      </footer>
    </Row>
  );
};

export default footer;
