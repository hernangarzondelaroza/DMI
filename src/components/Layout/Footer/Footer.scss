.footer {
  position: relative;
  flex-shrink: 0;
  text-align: center;
  background-color: #007bff;
  color: white;
}
