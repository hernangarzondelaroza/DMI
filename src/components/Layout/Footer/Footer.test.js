import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('Footer', () => {
  it('Should take a snapshot', () => {
    const component = renderer
      .create(<Footer />);
    expect(component).toMatchSnapshot();
  })
})