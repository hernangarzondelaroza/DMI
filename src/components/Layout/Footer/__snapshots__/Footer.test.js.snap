// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should take a snapshot 1`] = `
<div
  className="row"
>
  <footer
    className="footer fixed-bottom"
  >
    by Hernan Garzon de la Roza
  </footer>
</div>
`;
